src/content/agda/editor-note.tex

```diff
@@ -28,7 +28,7 @@
 The code is laid out in the following manner: the original Haskell code, followed by \optlang code.
 To distinguish between them, the code snippets are braced from the left with a vertical bar, in the primary
 color of the language's logo, \raisebox{-.2mm}{\includegraphics[height=.3cm]{fig/icons/haskell.png}},
-and \raisebox{-.2mm}{\includegraphics[height=.3cm]{\optlangfig}} respectively, e.g.:
+and \raisebox{-1mm}{\includegraphics[height=.4cm]{\optlangfig}} respectively, e.g.:
```

Comment:
Previously the Agda logo seemed to be floating to high above the line of text and seemed a bit smaller than the nearby Haskell logo.

Reply:
Nice!